plugin/core/types.py

```diff
@@ -507,7 +507,7 @@ def __init__(self,
                  binary_args: Optional[List[str]] = None,  # DEPRECATED
                  tcp_port: Optional[int] = None,
                  auto_complete_selector: Optional[str] = None,
-                 allow_completion_triggers_from_server: bool = True,
+                 ignore_server_triggers: bool = False,
```

Comment:
I think `ignore_server_triggers` is not clear what the "triggers" are. I think we need longer `ignore_server_trigger_chars`.